charts/cheetah-opensearchrole/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: cheetah-opensearchrole
+description: A Helm chart for creating opensearch roles with both read and write permissions based on a given index name.
```

Comment:
Hmm I am in doubt if it makes sense, as it would be a very strong dependency between index templates and roles... on the other hand, that is the exact same situation we are in right now. I would like to discuss it.

My initial thought was to have 2 index templates instead. Also due to access control, as developers should be able to define index templates mappings but not who have access to the data.

Reply:
Index templates was just an example - we could encode all sorts of stuff with this chart. But if you only want this chart to handle oauth2-usable roles/bindings, then it's all good